packages/google_maps_flutter/google_maps_flutter_ios/pigeons/messages.dart

```diff
@@ -357,6 +357,43 @@ class PlatformCameraTargetBounds {
   final PlatformLatLngBounds? bounds;
 }
 
+/// Pair of double values, such as for an offset or size.
+class PlatformDoublePair {
+  PlatformDoublePair(this.x, this.y);
+
+  final double x;
+  final double y;
+}
+
+/// Pigeon equivalent of the GroundOverlay class.
+class PlatformGroundOverlay {
+  PlatformGroundOverlay({
+    required this.groundOverlayId,
+    required this.image,
+    required this.position,
+    required this.bounds,
+    required this.anchor,
+    required this.transparency,
+    required this.bearing,
+    required this.zIndex,
+    required this.visible,
+    required this.clickable,
+    required this.zoomLevel,
+  });
+
+  final String groundOverlayId;
+  final PlatformBitmap image;
+  final PlatformLatLng? position;
+  final PlatformLatLngBounds? bounds;
+  final PlatformDoublePair? anchor;
```

Comment:
Isn't this conceptually a point? We already have `PlatformPoint`, so shouldn't need a new `PlatformDoublePair` for this unless there's a conceptual difference in meaning here.

Reply:
Changed this to PlatformPoint.